### PR TITLE
TranscoderSub.

### PR DESCRIPTION
**Pull request by j0sh**
For https://github.com/livepeer/go-livepeer-basicnet/issues/29

### CODE REVIEW

**Comment by j0sh** [comment]
Rebased to latest goclient master, tests fixed.

**Comment by j0sh** [comment]
Made bind address configurable to accommodate latest basicnet changes. Transcoders will need to be explicit about the addresses they bind to for now; libp2p will initiate outgoing connections with just the first one.

File: cmd/livepeer/livepeer.go [hunk 1]
```diff
@@ -148,8 +150,29 @@ func main() {
...
+		mas := strings.Split(*bindIPs, ",")
+		for _, m := range mas {
+			addr, err := ma.NewMultiaddr(m)
+			if err != nil {
+				glog.Errorf("Error creating bindIP %v to multiaddr: %v", m, err)
+				continue // nonfatal
+			}
+			maddrs = append(maddrs, addr)
+		}
+	} else if *transcoder {
```
Comment (j0sh) [inline cmd/livepeer/livepeer.go]: This bit is for after libp2p fixes their issue then transcoders can just have the correct IPs auto-selected.

**Review by ericxtang — APPROVED** [review]
Let's merge this.  🚢

**Pull request merged**